src/components/ImageViewer/ImageViewer.scss

```diff
@@ -1,8 +1,11 @@
 // TODO: all colors should be using variables and inheriting from sitebranding
 .image-border{
-  background-color: #f2f8fb
+  background-color: #f2f8fb;
+  display: inline-block;
+  min-width: 358px;
```

Comment:
Is this necessary? If .image-title doesn't get smaller than 350px and it has 4px padding, isn't this implied?

Reply:
With out that line the blue/grey box scaled to the width of the image not the text so the text was wider then the box.  But if there is a better way to do this let me know!